scripts/repo.json

```diff
@@ -49,25 +47,25 @@
         "name": "gemstone",
         "type": "git",
         "url": "https://github.com/StanfordAHA/gemstone",
-        "branch": "master"
+        "branch": "magma2"
     },
     {
         "name": "canal",
         "type": "git",
         "url": "https://github.com/StanfordAHA/canal",
-        "branch": "master"
+        "branch": "magma2"
     },
     {
         "name": "peak",
         "type": "git",
         "url": "https://github.com/phanrahan/peak",
-        "branch": "e958ab4996a49ff19d2c2ed5bdfcffc02a69eb36"
+        "branch": "update-rtl-utils"
     },
     {
         "name": "lassen",
         "type": "git",
         "url": "https://github.com/StanfordAHA/lassen",
-        "branch": "b81035a79182fb075e11dac11c45b2ad1d3d218e"
+        "branch": "fp-mask"
```

Comment:
Is fp-mask still correct?

Reply:
Yes: https://github.com/StanfordAHA/lassen/pull/157 (we changed the name of the PR but kept the branch)